pkgs/development/python-modules/pytmx/default.nix

```diff
@@ -1,21 +1,25 @@
-{ lib, fetchFromGitHub, isPy3k, buildPythonPackage, pygame, pyglet, pysdl2, six }:
+{ lib, fetchFromGitHub, isPy27, buildPythonPackage, pygame, pyglet, pysdl2, six }:
 
 buildPythonPackage rec {
   pname = "pytmx";
-  version = "3.22.0";
+  version = "3.24.0";
+
+  disabled = isPy27;
 
   src = fetchFromGitHub {
     # The release was not git tagged.
     owner = "bitcraft";
     repo = "PyTMX";
-    rev = "187fd429dadcdc5828e78e6748a983aa1434e4d2";
-    sha256 = "0480pr61v54bwdyzb983sk0fqkyfbcgrdn8k11yf1yck4zb119gc";
+    rev = "eb96efea30d57b731654b2a167d86b8b553b147d";
+    sha256 = "1g1j4w75zw76p5f8m5v0hdigdlva2flf0ngyk8nvqcwzcl5vq5wc";
   };
 
   propagatedBuildInputs = [ pygame pyglet pysdl2 six ];
 
   checkPhase = ''
-    python -m unittest tests.pytmx.test_pytmx
+    # Change into the test directory due to a relative resource path.
+    cd tests/pytmx
```

Comment:
Such stuff normally belongs into preCheck but we are overwriting checkPhase so it should be fine. Can we use pytestCheckHook here?

Reply:
Thanks for your feedback. I tried the `pytestCheckHook` thingy, but was not able to run the test from within the test directory. Switching directories within `preCheck` seems to confuse `pytest`.

Thus, I applied your other suggestions.

Reply:
> I tried the `pytestCheckHook` thingy

It is a hook not a thingy.